### PR TITLE
Ghost roles: rule stuff

### PR DESCRIPTION
**Pull request by 20kdc**
## About the PR
See screenshots

**Screenshots**
image

**Changelog**

:cl:
- tweak: Expanded default ghost role rules and removed excuses to not see them by adding a 5-second delay

### CODE REVIEW

File: Content.Client/Ghost/Roles/UI/GhostRoleRulesWindow.xaml.cs [hunk 1]
```diff
@@ -4,17 +4,34 @@
...
+        {
+            base.FrameUpdate(args);
+            if (_timer > 0.0)
+            {
+                _timer -= args.DeltaSeconds;
+            }
+            else
+            {
+                RequestButton.Disabled = false;
+            }
```
Comment (PaulRitter) [inline Content.Client/Ghost/Roles/UI/GhostRoleRulesWindow.xaml.cs:34]:
```suggestion
            base.FrameUpdate(args);
            if (!RequestButton.Disabled) return;
            if (_timer > 0.0)
            {
                _timer -= args.DeltaSeconds;
            }
            else
            {
                RequestButton.Disabled = false;
            }
```

File: Resources/Locale/en-US/ghost/roles/ghost-role-component.ftl [hunk 2]
```diff
@@ -1,3 +1,6 @@
 # also used in MakeGhostRuleWindow and MakeGhostRoleCommand
 ghost-role-component-default-rules = You don't remember any of your previous life unless an administrator tells you otherwise.
+                                     At all. [color=red]In any way.[/color]
+                                     You are allowed to remember knowledge about the game in general, such as how to cook, how to use objects, etc.
+                                     You are absolutely [color=red]NOT[/color] allowed to remember, say, the name, appearance, etc. of your previous character.
```
Comment (PaulRitter) [inline Resources/Locale/en-US/ghost/roles/ghost-role-component.ftl:5]:
```suggestion
                                     You are allowed to remember knowledge about the game in general, such as how to cook, how to use objects, etc.
                                     You are absolutely [color=red]NOT[/color] allowed to remember, say, the name, appearance, etc. of your previous character.
```
Honestly, im not really even sure we need to further specify this. the 5 second cooldown on the button should suffic imo

**Review by PaulRitter — CHANGES_REQUESTED** [review]

File: Content.Client/Ghost/Roles/UI/GhostRoleRulesWindow.xaml.cs (hunk 1 above)
Comment (20kdc) [reply Content.Client/Ghost/Roles/UI/GhostRoleRulesWindow.xaml.cs]: Applied

File: Resources/Locale/en-US/ghost/roles/ghost-role-component.ftl (hunk 2 above)
Comment (20kdc) [reply Resources/Locale/en-US/ghost/roles/ghost-role-component.ftl]: Applied

**Comment by 20kdc** [comment]
Suggestions applied

**Pull request merged**